tensorflow_addons/callbacks/tqdm_progress_bar.py

```diff
@@ -93,11 +93,17 @@ def __init__(
         self.last_update_time = time.time()
         self.overall_progress_tqdm = None
         self.epoch_progress_tqdm = None
+        self.is_training = False
+        self.test_last_update_time = time.time()
```

Comment:
Are those `test_*` variables necessary anymore? Since we are not going to display the progress bar for `train with validation_data`, can we re-use the same variable for `train()` and `evaluate()`? This would make it easier to maintain, thanks!

Reply:
Also since things should be closed properly in `on_*_end` and opend properly at `on_*_begin`, I think it would be possible to re-use those variables.